src/iarc7_motion/iarc_tasks/task_utilities/obstacle_avoid_helper.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
Same comments as for idle obstacle avoider

Reply:
Also can we make this a part of the task topic buffer in AbstractTask so we don't need to create a new subscriber for every task?